kernel/network/e1000.c

```diff
@@ -3,54 +3,58 @@
 #include <stdio.h>
 #include "kernel/pci/pci.h"
 #include "kernel/pci/pci_mmio.h"
+#include "kernel/mmio.h"
+#include "kernel/logger/logger.h"
```

Comment:
I think we should remove the logger when merging in our master

Reply:
Why do you want to remove it completely?

Reply:
I would just remove it from this file because the debug messages are rather for us during development than for the user of the OS